src/app/organizer/DropDown.tsx

```diff
@@ -25,30 +32,117 @@ function MenuItem({ item }: { item: DropDownItem }) {
   );
 }
 
+function Menu({ items }: { items: DropDownItem[] }) {
+  return (
+    <div className={styles.menu}>
+      {items.map((item) => (
+        <MenuItem key={item.id} item={item} />
+      ))}
+    </div>
+  );
+}
+
+function OrderedMenuItem({ item, index }: { item: DropDownItem; index: number }) {
+  const { checked, content, onItemSelect } = item;
+
+  return (
+    <Draggable draggableId={item.id} index={index}>
+      {(provided) => (
+        <div
+          className={`${styles.orderedMenuItem} ${item.checked ? '' : ' disabled'}`}
+          data-index={index}
+          ref={provided.innerRef}
+          {...provided.draggableProps}
+          onClick={onItemSelect}
+        >
+          <span className={styles.dragHandle} {...provided.dragHandleProps}>
+            <AppIcon icon={reorderIcon} className="reorder-handle" />
+          </span>
+          <span className={styles.name}>{content}</span>
+          <span className={styles.checkIcon}>
+            <AppIcon
+              icon={checked ? enabledIcon : unselectedCheckIcon}
+              className="checked-toggle-icon"
+            />
+          </span>
+        </div>
+      )}
+    </Draggable>
+  );
+}
+
+function OrderedMenu({
```

Comment:
It's best to break all these up into their own files, unless they're really internal details. In this case it feels like this DropDown file has gotten too complicated.

Also, as we discussed on Slack, I think making the columns reorderable should probably be left as a followup feature.